packages/plans-grid-next/src/hooks/use-grid-size.ts

```diff
@@ -16,37 +16,31 @@ interface Props {
  * useGridSize returns the current grid size based on the width of the container
  * and the breakpoints passed through as props.
  */
-const useGridSize = ( { containerRef, containerBreakpoints }: Props ) => {
+export default function useGridSize( { containerRef, containerBreakpoints }: Props ) {
```

Comment:
`Non-blocking`: 

Out of curiosity, was there a reason we switched to a function declaration?

Reply:
My intent was to do the default export together with the function definition, and still give the function a name. This doesn't work:
```js
export default const useGridSize = () => ...
```
You can only export the function expression itself, but then it has no name:
```js
export default () => ...
```
It's mostly personal preference to define functions this way, nothing very objective.

Reply:
Some prefer them because of how hoisting works for `function` declarations, and it doesn't for `const` ones. But hoisting doesn't matter in this instance. Inline name is a good argument though 👍